src/group_theory/submonoid/membership.lean

```diff
@@ -334,6 +334,16 @@ lemma mul_left_mem_add_closure (ha : a ∈ S) (hb : b ∈ add_submonoid.closure
   a * b ∈ add_submonoid.closure (S : set R) :=
 S.mul_mem_add_closure (add_submonoid.mem_closure.mpr (λ sT hT, hT ha)) hb
 
+/-- An element is in the closure of a two-element set if it is a linear combination of those two
+elements. -/
+@[to_additive]
+lemma mem_closure_pair {A : Type*} [comm_monoid A] (a b c : A) :
```

Comment:
Can you please add the same statement for groups and modules? Thanks!

Reply:
In which file would these go? The method group.closure is a directory marked "deprecated", and documentation says its contents should not be used for any new projects

Reply:
I don't think I'll be able to do module closures, that can be another project I guess

Reply:
The group one will go in `group_theory/subgroup`